pages/agent/v3/aws/autoscaling_mac_metal.md.erb

```diff
@@ -0,0 +1,200 @@
+
+In supported regions, `mac1.metal` dedicated hosts may not be available in
+every Availability Zone. Consider using a subnet in every Availability Zone to
+maximise the pool of instances available to boot from.
+
+You also need to configure or define the VPC Security Groups your instance
+network interfaces should belong to. At a minimum, inbound SSH access is
+required to set up your initial template AMI.
+
+## Build an AMI
+
+Before deploying this template, you must create a template AMI that can be
+horizontally scaled across multiple instances.
+
+1. Reserve a [AWS mac1.metal](https://aws.amazon.com/ec2/instance-types/mac/)
+Dedicated Host.
+1. Boot a macOS instance using your desired AMI on the Dedicated Host.
+1. Configure instance VPC subnets, security groups, and key pairs so that you
+can access the instance.
+1. Using an SSH or SSM session:
+	* Set a password for the ec2-user using `sudo passwd ec2-user`
+	* Enable screen sharing using `sudo /System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/Resources/kickstart -activate -configure -access -on -restart -agent -privs -all`
+1. Using a VNC session:
+	1. Sign in as the `ec2-user`
+	1. Enable *Automatic login* for the ec2-user in *System Preferences* > *Users & Accounts* > *Login Options*
+	1. Disable *Require password* in *System Preferences* > *Security & Privacy* > *General*
+1. Install your required version of Xcode, and ensure you launch Xcode at least
+once so you are presented with the EULA prompt.
+1. Using the AWS EC2 Console, create an AMI from your instance.
+
+You do not need to install the `buildkite-agent`, it is automatically installed
+by the Launch Template `UserData` script.
+
+## Associate your AMI with a Customer managed license in AWS License Manager
+
+To launch an instances using a Host Resource Group, the instance AMI must be
+associated with a Customer managed license in *AWS License Manager*.
+
+Using the AWS Console, open the *AWS License Manager* and navigate to
+*Customer managed licenses*. Create a new Customer managed license, enter a
+descriptive name and select a License Type of `Cores`.
+
+Once your Customer managed license has been saved, open the detail view for your
+license. Open the *Associated AMIs* tab and choose *Associate AMI*. From the
+list of *Available AMIs*, select your macOS AMIs and then click *Associate*.
+
+## Deploy the CloudFormation template
+
+Using the VPC and AMI prepared earlier, prepare values for the following
+required parameters:
+
+* `ImageId` from your AMI set up
+* `RootVolumeSize`
+* `Subnets` from your VPC set up
+* `SecurityGroupIds` from your VPC set up
+* `IamInstanceProfile` if accessing AWS services from your builds, provide an Instance Profile ARN with an appropriate IAM role attached
+* `BuildkiteAgentToken` an Agent Token for your [Buildkite Organization](http://buildkite.com/organizations/-/agents)
+* `BuildkiteAgentQueue` the Buildkite Queue your Pipeline Steps target
```

Comment:
```suggestion
* `BuildkiteAgentQueue` the Buildkite queue your pipeline steps target
```

Reply:
Should this be the Buildkite queue of your pipeline steps target? I don't quite follow what it means with the current phrasing

Reply:
```suggestion
* `BuildkiteAgentQueue` the Buildkite Queue your Pipeline Steps target
* `BuildkiteAgentQueue` the Buildkite Queue your pipeline steps use
```

?